commands/set_channel.js

```diff
@@ -18,13 +18,29 @@ module.exports = {
             },
         });
         
-        // TODO: Check if the bot can access the channel or not.
-        
         if (query) {
             await interaction.reply({ content: 'This channel is already set as the channel', ephemeral: true });
             return;
         }
-
+        
+        try {
+            const startEmbed = new EmbedBuilder()
+			.setColor(0xed1c24) // TODO: Change to different color.
```

Comment:
This is "the color of the bot", I choose this because it's the most dominant on the new logo I made.